lua/config/options.lua

```diff
@@ -15,7 +15,7 @@ vim.opt.mousescroll = "ver:1,hor:6" -- Default: "ver:3,hor:6"
 vim.opt.wrap = true
 vim.opt.linebreak = true
 vim.opt.breakindent = true
-
+vim.keymap.set("i", "<m-bs>", "<c-w>", { noremap = true, silent = true })
```

Comment:
I think this will be fine for me now that I think about it because alacritty makes it so that whenever I press `<option+bs>` it's actually just inputting `<C-w>`, so this keymap will have no effect on me since neovim never actually gets a `<m-bs>` input from me.

Before merging though move this to the `keymaps.lua` file. All keymaps are put in there. This options file is just for when ur setting options with `vim.opt...` or something similar